### PR TITLE
add linked table. #15

### LINKED ISSUE #58: Show AE listing when a point is clicked

### PR DESCRIPTION
**Pull request by jwildfire**
Show a basic linked table when a point is clicked. A bit buggy since it's showing all AEs in the clicked strata - even for groups that aren't selected. Probably ok for now. 

Run `volcanoApp()` to test. 

fix #58

### CODE REVIEW

File: R/volcanoPlot.R [hunk 1]
```diff
@@ -77,7 +77,21 @@ volcanoPlot <- function(data, plotly = TRUE, ...){
                                 showarrow = F, xref = 'paper', yref = 'paper',
                                 xanchor = 'right', yanchor = 'bottom', xshift = 0, yshift = 0,
                                 font = list(size = 12, color = "blue")))
-             )
+             ) %>%
+          htmlwidgets::onRender("
+          console.log('hi mom')
```
Comment (anhtr) [inline R/volcanoPlot.R:82]: I'm not sure if this is a placeholder or not, but it is funny and doesn't seem to affect the final result. :-)

File: R/volcano_server.R [hunk 2]
```diff
@@ -38,15 +38,54 @@ volcano_server <- function(input, output, session, params) {
...
+        #dm <- params()$data$dm
+        #ids <- dm[dm[params()$settings$dm$treatment_col] %in% groups(),]%>%
+        #    pull(params()$settings$dm$id_col)
+        #print(ids)
+        #sub2<-sub1[sub1[params()$settings$dm$id_col,] %in% ids,]
+
+        sub1
+    })
+
+    output$click <- renderText({
```
Comment (anhtr) [inline R/volcano_server.R:81]: I'm not sure why, but at least on my computer this sentence can't not go longer than the width of the plot.

image

image

**Review by anhtr — APPROVED** [review]

File: R/volcanoPlot.R (hunk 1 above)
Comment (jwildfire) [reply R/volcanoPlot.R:82]: Oh yeah - was trying out a different approach before I figured out `event_data()` thing. Will delete this.

**Review by anhtr — APPROVED** [review]

**Comment by jwildfire** [comment]
Not sure how to tie the point clicked all the way back to the AE listing ... need to dig in to `plotly::event_data` a bit more.

**Comment by nandriychuk** [comment]
To summarize the code state in this PR right now since multiple people worked on it at different times:
- The ``plotly_click`` is now working correctly and points to the correct strata when clicked. 
- The ae_sub reactive is where madness lies:
   ``sub1`` and ``ids`` objects produce data upon ``plotly_click``, however ``sub2`` results in an empty dataset when ``sub1`` is filtered using the ``ids`` object. 

Note, I left the print statements as well as Jeremy's initial code commented out intentionally per our discussion during the last call.

**Pull request merged**